3rdparty/nuxeo/nuxeo-platform-elasticsearch/src/main/java/org/collectionspace/services/nuxeo/elasticsearch/DefaultESDocumentWriter.java

```diff
@@ -190,6 +191,37 @@ private void denormExhibitionRecords(CoreSession session, String csid, String te
 	denormValues.putArray("exhibition").addAll(exhibitions);
 }
 
+	/**
+	 * Denormalize the material group list for a collectionobject in order to index the controlled or uncontrolled term
+	 *
+	 * @param doc the collectionobject document
+	 * @param denormValues the json node for denormalized fields
+	 */
+	private void denormMaterialRecords(DocumentModel doc, ObjectNode denormValues) {
```

Comment:
I think this should be called `denormMaterialFields`, as it's just combining fields into one, rather than pulling fields from related records.